app/models/ontology.rb

```diff
@@ -256,12 +253,11 @@ def file_in_repository
 
   protected
 
-  def import_links
-    Link.where(source_id: self.id, kind: "import")
+  def import_mappings
+    Mapping.where(source_id: id, kind: "import")
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.